### PR TITLE
Oh my, look at the time, it's time for another UPSTREAM MERGE DUE TO A DOZEN MIRRORS!

### LINKED ISSUE #5409: Airlock Sprites & Locker Sprites
The airlocks for the general access hallways are a default green, revert to original.
Our locker sprites were changed, revert to original.

### PR DESCRIPTION
**Pull request by ghost**
Closes #5413
Closes #5412 
Closes #5410 
Closes #5407 
Closes #5406 
Closes #5402 
Closes #5400 
Closes #5392 
Closes #5391 
Closes #5379 
Closes #5377 
Closes #5370 
Closes #5369 
Closes #5409

### CODE REVIEW

**Comment by ghost** [comment]
Please recompile TGUI and update the changelog

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
-[Directions]: # (INCLUDE THE ROUND ID from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a test merge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. Explain your issue in detail, including the steps to reproduce it.)
+[Directions]: # (If you discovered this issue from playing tgstation hosted servers
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE.md]: > tgstation hosted servers

**Comment by MoreRobustThanYou** [comment]
@Chnkr what changelog needs to be updated? I just updated the code we use to load our custom config.

**Comment by ghost** [comment]
If you can, please manually check all the mirrored files/procs if they need to be updated

**Comment by ghost** [comment]
@Chnkr I don't know which files we mirror

**Comment by ghost** [comment]
Already recompiled changelog

**Comment by ghost** [comment]
@Chnkr I recompiled tgui and git detected no changes. I didn't see any tgui changes to .ract files in the commits, so it's fine

File: html/changelog.html [hunk 2]
```diff
@@ -59,6 +59,64 @@
...
+			<h3 class="author">Xhuis updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">Grinding runed metal and brass now produces iron/blood and iron/teslium, respectively.</li>
+				<li class="balance">As part of some code-side improvements, the amount of reagents you get from grinding some objects might be slightly different.</li>
+				<li class="bugfix">Some grinding recipes that didn't work, like dead mice and glowsticks, now do.</li>
+				<li class="bugfix">All-In-One grinders now correctly grind up everything, instead of one thing at a time.</li>
+			</ul>
+			<h3 class="author">nicbn updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="imageadd">Closet sprites changed.</li>
```
Comment (Manly-Dwarf) [inline html/changelog.html:117]: no
no new closets

**Review by Manly-Dwarf — CHANGES_REQUESTED** [review]

**Comment by Manly-Dwarf** [comment]
good boy removed the closets

**Comment by ghost** [comment]
@Kawaii-Big-Boss die in a hole

**Review by ghost — DISMISSED** [review]

**Review by Manly-Dwarf — APPROVED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**